src/account/mod.rs

```diff
@@ -372,6 +373,21 @@ impl AccountHandle {
     }
 }
 
+impl Drop for Account {
+    fn drop(&mut self) {
+        if self.has_pending_changes {
+            let storage_path = self.storage_path.clone();
+            let account_id = self.id().clone();
+            if let Ok(data) = serde_json::to_string(&self) {
+                let _ = thread::spawn(move || {
```

Comment:
Why not spawn a task instead of a full thread ?

Reply:
I need the thread here to use my tokio runtime :(